src/RequestMetadata.php

```diff
@@ -4,29 +4,60 @@
 
 namespace IronCore;
 
+/**
+ * Holds metadata fields as part of an encrypted document. Each encrypted document will have
+ * metadata that associates it to a tenant ID, which service is accessing the data,
+ * as well as optional fields for other arbitrary key/value pairs and a request ID
+ * to send to the Tenant Security Proxy.
+ */
 class RequestMetadata
 {
-    private $tenant_id;
-    private $icl_fields;
-    private $custom_fields;
+    /**
+     * @var string Unique ID of tenant that is performing the operation
```

Comment:
Nitpick: The tenant isn't performing the action, the action is being performed for a particular tenant.

Reply:
Fixed. We should maybe change this in our Java SDK as well